### PR TITLE
Refactor to use ES6

### PR DESCRIPTION
**Pull request by kelvinbush**
In this pull request I did the following:

- Divided the code into modules.
- Used ES6 syntax with modules.
- Added [luxon](https://moment.github.io/luxon) library for the date.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    .map((book, index) => `
+        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
+            <div>
+                <p class="title">"${book.title}" by ${book.author}</p>
+            </div>
+            <button data-id=${book.id} class="remove">Remove</button>
+        </article>`)
+    .join('');
+}
+
```
Comment (ShoiraTa) [inline index.js]: - [ ] Kindly use arrow function:
```
const renderBookList = () => {
 bookListSection.innerHTML = awesomeBooks.getBooks()
    .map((book, index) => `
        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
            <div>
                <p class="title">"${book.title}" by ${book.author}</p>
            </div>
            <button data-id=${book.id} class="remove">Remove</button>
        </article>`)
    .join('');
}
```

File: README.md [hunk 2]
```diff
@@ -62,11 +50,6 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
```
Comment (ShoiraTa) [inline README.md]: - [ ] This section has no License. Kindly delete this section or add a License file with link. I strongly recommend to add .MIT License to the project root and add a link to it here 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @kelvinbush ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- Properly using ES6 and modules ✔️ 
- Professional project setup! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Kelvin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**